### ISSUE TITLE
How long to train the model from scratch

### ISSUE THREAD

**Issue by valencebond** [ashual/scene_generation#8]
i train the model nealy 24 hours，but only get one tenth of the train stage ` t = 108800 / 1000000`，my device is one TITAN Xp.

**Comment by ghost** [comment]
Yes, it will take about 160 hours to train in a single GPU(in my case, 2080).
BTW, you can try to adjust **image_print_every** to reduce training time.
